app/src/main/java/com/gathering/android/event/eventdetail/EventDetailScreen.kt

```diff
@@ -114,4 +179,171 @@ class EventDetailScreen : Fragment(), EventDetailNavigator {
             bundle
         )
     }
+
+    @Preview(showBackground = true)
+    @Composable
+    fun EventDetailPreview() {
+        EventDetail(
+            eventName = "Ani",
+            eventHostEmail = "animan@gmail.com",
+            description = "party",
+            photoUrl = "",
+            address = "101 Erskine Ave",
+            date = "15 Nov 2023",
+            time = "15:30",
+            acceptType = AcceptType.No,
+            {},
+            {},
+            {},
+            {}
+        )
+    }
+
+    @Composable
+    fun EventDetail(
+        eventName: String,
+        eventHostEmail: String,
+        description: String,
+        photoUrl: String,
+        address: String,
+        date: String,
+        time: String,
+        acceptType: AcceptType,
+        onYesButtonClick: () -> Unit,
+        onNoButtonClick: () -> Unit,
+        onMaybeButtonClick: () -> Unit,
+        onAttachListClicked: () -> Unit,
+    ) {
+        Column(
+            Modifier
+                .padding(10.dp)
+                .fillMaxSize()
+        ) {
+            EventImage(photoUrl)
+            Text(
+                fontSize = 20.sp,
+                fontWeight = FontWeight.Bold,
+                text = eventName,
+                modifier = Modifier.padding(start = 10.dp, 40.dp)
+            )
+            Text(
```

Comment:
i suggest we make these composables:

@Composable
private fun **EventInfo**(
    eventName: String,
    eventHostEmail: String,
    description: String,
    address: String,
    date: String,
    time: String
) {
    EventTitle(eventName)
    TextInfo(eventHostEmail)
    TextInfo(description)
    TextInfo(address)
    TextInfo(date)
    TextInfo(time)
}

@Composable
fun **EventTitle**(eventName: String) {
    Text(
        fontSize = 20.sp,
        fontWeight = FontWeight.Bold,
        text = eventName,
        modifier = Modifier.padding(start = 10.dp, 40.dp)
    )
}

@Composable
fun **TextInfo**(text: String) {
    Text(
        fontSize = 16.sp,
        text = text,
        modifier = Modifier.padding(10.dp)
    )
}

and call it in EventDetail like so instead of having 5 different Texts:

EventInfo(
            eventName,
            eventHostEmail,
            description,
            address,
            date,
            time
        )

Reply:
I totally agree,  it would be better to do that.